index.html

```diff
@@ -3,16 +3,22 @@
   <head>
     <title>"ALL about me"</title>
```

Comment:
The quotation marks around the title can be removed. Your title will show up without them.